webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -132,25 +114,34 @@ const HostDetails = ({ match, location: { hash } }) => {
           </Text>
           <br />
         </div>
-        <Tabs
-          style={{
-            width: window.innerWidth - (isNavCollapsed ? 95 : 220),
-          }}
-          activeKey={activeTab}
-          onSelect={handleTabClick}
-        >
-          {tabs &&
-            tabs.map(tab => (
-              <Tab eventKey={tab} title={tab}>
-                <Slot
-                  response={response}
-                  status={status}
-                  id="host-details-page-tabs"
-                  fillID={tab}
-                />
-              </Tab>
-            ))}
-        </Tabs>
+        <HashRouter>
+          <>
+            <Tabs
+              style={{
+                width: window.innerWidth - (isNavCollapsed ? 95 : 220),
+              }}
+              activeKey={hash.slice(2)}
+            >
+              {tabs &&
+                tabs.map(tab => (
+                  <Tab eventKey={tab} title={tab} href={`#${tab}`} />
+                ))}
```

Comment:
in the PF4 implementation, the tab's body is the children passed to it,
should we pass the Route and slot inside it?

Reply:
Thanks @laviro! I was thinking that we need a `<Switch>` for wrapping multiple routes, but it works well.